File: C++/level 4/Representation of numbers.cpp
#include<iostream>
using namespace std;
int expressions(int testCase)
{
	int answer = 1;
	int tem = 0;
  for(int a=1; a<=testCase/2; a++){
  	tem = a;
    for(int b=a+1; b<=testCase/2+1; b++){
      tem+=b;
    	if(tem==testCase){
        answer++;
      	break;
      }
    }
  }
	return answer;
}

int main()
{
	int testNo = 15;
	int testAnswer = expressions(testNo);
// 아래는 테스트로 출력해 보기 위한 코드입니다.
	cout<<testAnswer;
}

File: Python/level 4/Representation of numbers.py
def expressions(num):
    answer = 1
    tem = 1
    for a in range(1,num//2+1):
        tem = a
        for b in range(a+1,num//2+2):
            tem += b
            if tem == num:
                answer += 1
                break
    return answer


# 아래는 테스트로 출력해 보기 위한 코드입니다.
print(expressions(15))
